spaceship/spec/test_flight/app_test_info_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'spec_helper'
+
+describe Spaceship::TestFlight::AppTestInfo do
+  let(:app_test_info) do
+    Spaceship::TestFlight::AppTestInfo.new({
+                                            'primaryLocale' => 'en-US',
+                                            'details' => [{
+                                              'locale' => 'en-US',
+                                              'feedbackEmail' => 'feedback@email.com',
+                                              'description' => 'Beta app description',
+                                              'whatsNew' => 'What is new'
+                                            }],
+                                            'betaReviewInfo' => {
+                                              'contactFirstName' => 'First',
+                                              'contactLastName' => 'Last',
+                                              'contactPhone' => '1234567890',
+                                              'contactEmail' => 'contact@email.com',
+                                              'demoAccountName' => 'User Name',
+                                              'demoAccountPassword' => 'Password',
+                                              'demoAccountRequired' => false,
+                                              'notes' => 'notes!!'
+                                            }
```

Comment:
Could the pilot docs please be updated to show an example of adding these settings to your `Fastfile` (or however they meant to be set). Thanks.